### ISSUE TITLE
Functions should be static instead of instance-based.

### ISSUE THREAD

**Issue by JoshMcCullough** [simogeo/geostats#22]
For most functions, it seems I should be able to simply call (for example), `geostats.min(values)` without creating a `geostats` instance.

I haven't used the library enough yet to understand when/why I'd want my own instance of `geostats` instead of just calling its functions statically.

**Comment by simogeo** [comment]
This could be discussed indeed but you could also imagine to compare several series as follow : 

```javascript
if(serie1.min() < serie2.min()) {
// whatever
}
```

**Comment by JoshMcCullough** [comment]
Yes, I think it is pretty simple to make it work both ways but having the instance methods call the static methods. Just a suggestion. :+1:

**Comment by simogeo** [comment]
Would you propose a pull ?

**Comment by JoshMcCullough** [comment]
Yes I'll work on it at some point this week.

**Comment by mtralka** [comment]
@JoshMcCullough Apologies to ping an old issue. I've made [GeoBuckets](https://github.com/mtralka/GeoBuckets) which is a modern take on geostats. All functions are static. Additionally the library is tree-shakable and asynchronous. Please let me know if that fulfills your usage scenario. If not, I'd be happy to accept PRs or make adjustments as needed

**Comment by simogeo** [comment]
@mtralka see new reference to GeoBuckets in _geostats_ readme file 418113a048d32d49b34b9f2694122f392aa0f5ad

**Comment by mtralka** [comment]
@simogeo Awesome stuff! I appreciate the mention. Thank you for your continued work with geostats
